glib-sys/Cargo.toml

```diff
@@ -32,7 +32,7 @@ description = "FFI bindings to libglib-2.0"
 homepage = "http://gtk-rs.org/"
 keywords = ["glib", "ffi", "gtk-rs", "gnome"]
 license = "MIT"
-links = "glib"
+links = "glib-2.0"
 name = "glib-sys"
 repository = "https://github.com/gtk-rs/sys"
 version = "0.10.0"
```

Comment:
A bit below, we also need to duplicate the `package.metadata.system-deps` blocks: we also need to pull in `gobject-2.0` in addition to `glib-2.0` as all the `get_type()` functions are actually from `gobject-2.0`.

Not going to be a problem in practice because it's unlikely that `glib-sys` is used without `gobject-sys` but would be more correct.

Reply:
updated :)